### PR TITLE
Message controller

### PR DESCRIPTION
**Pull request by dracos**
The first commit I think is safe and could be taken alone; am working on second commit more/testing needed, but hopefully along the right lines, I think it could sort itself out better but I tried for now to keep the business code as similar as possible, maybe. [skip changelog]

### CODE REVIEW

File: web/cobrands/buckinghamshire/assets.js [hunk 1]
```diff
@@ -352,54 +312,31 @@ fixmystreet.assets.add($.extend(true, {}, defaults, {
     all_categories: true,
     actions: {
         found: function(layer, feature) {
-            fixmystreet.body_overrides.allow_send(layer.fixmystreet.body);
-            fixmystreet.body_overrides.remove_only_send();
+            var map = {
+                "HE": '#js-not-council-road-he',
+                "HWOA": '#js-not-council-road-other'
+            };
+            var bad_id = map[feature.attributes.feature_ty] || '#js-not-council-road';
```
Comment (struan) [inline web/cobrands/buckinghamshire/assets.js]: I am not sure about bad_id as a name. msg_id would make more sense?

File: web/cobrands/northamptonshire/assets.js [hunk 2]
```diff
@@ -617,49 +617,21 @@ function ncc_is_emergency_category() {
     return {relevant: relevant, currently_shown: currently_shown, body: body};
 }
```
Comment (struan) [inline web/cobrands/northamptonshire/assets.js]: Presumably this should be removed in this commit and not the next.

Comment (dracos) [reply web/cobrands/northamptonshire/assets.js]: Can't easily tell what this is referring to sorry; if it's ncc_is_emergency_category, that is still called somewhere, e.g. asset_found, until the last commit of this PR (when it's removed)?

Comment (struan) [reply web/cobrands/northamptonshire/assets.js]: Yes, so it is. I missed that.

**Pull request merged**